app/src/main/res/layout/expanded_results_box.xml

```diff
@@ -119,7 +119,10 @@
             android:layout_height="wrap_content"
             android:layout_weight="1"
             android:popupBackground="#FFF"
-            android:gravity="right"/>
+            android:gravity="right"
+            android:layout_marginRight="-8sp"
+            android:layout_marginTop="-8sp"
+            android:layout_marginBottom="-8sp" />
```

Comment:
Think I'm gonna revert this—doesn't work for me, names get cut. We'll have to see about this separately. @DJCrapsody